### PR TITLE
Update fast-install-and-first-run.rst

### PR DESCRIPTION
**Pull request by secnum**
Add IVRE Python 3 installation, external programs installation and minor corrections.

### CODE REVIEW

File: doc/install/fast-install-and-first-run.rst [hunk 1]
```diff
@@ -23,6 +23,21 @@ For another way to run IVRE easily (probably even more easily), see
 
 Install
 -------
+NB: if you are running Debian stable, the dokuwiki package has been
```
Comment (p-l-) [inline doc/install/fast-install-and-first-run.rst]: I **think** this is no longer true. Actually I have forgotten to update this page when I fixed the Docker image in c1c94ce1654d2b20fe47b45e471b8c81318d5cd6.

File: doc/install/fast-install-and-first-run.rst [hunk 2]
```diff
@@ -32,22 +47,24 @@ Install
    $ cd ivre
    $ python setup.py build
    $ sudo python setup.py install
+   $ echo 'DB_FLOW = "neo4j://neo4j:<new password>@localhost:7474/"' | sudo tee /etc/ivre.conf
```
Comment (p-l-) [inline doc/install/fast-install-and-first-run.rst]: This is going to be unnecessary very soon (#685).

File: doc/install/fast-install-and-first-run.rst [hunk 3]
```diff
@@ -74,6 +91,7 @@ Setup
    # echo '</Location>' >> conf-enabled/ivre.conf
    # sed -i 's/^\(\s*\)#Rewrite/\1Rewrite/' /etc/dokuwiki/apache.conf
    # echo 'WEB_GET_NOTEPAD_PAGES = "localdokuwiki"' >> /etc/ivre.conf
+   # chmod o+r /usr/local/share/ivre/web/wsgi/app.wsgi
```
Comment (p-l-) [inline doc/install/fast-install-and-first-run.rst]: When & why is that needed?

File: doc/install/fast-install-and-first-run.rst [hunk 4]
```diff
@@ -94,7 +112,58 @@ Database init, data download & importation
    $ sudo ivre ipdata --download --import-all
 
 The two last step may take a long time to run (40 minutes on a decent
-server), nothing to worry about.
+server), nothing to worry about. To initiate flowcli, Neo4j default
```
Comment (p-l-) [inline doc/install/fast-install-and-first-run.rst]: This is going to be unnecessary very soon (#685).

File: doc/install/fast-install-and-first-run.rst [hunk 5]
```diff
@@ -94,7 +112,58 @@ Database init, data download & importation
...
+- Nmap
+- Zeek/Bro
+- Argus
+- Nfdump
+- Neo4j
+
+::
+
+   $ sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 9DA31620334BD75D9DCB49F368818C72E52529D4
+   $ echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.0.list
```
Comment (p-l-) [inline doc/install/fast-install-and-first-run.rst]: Is this going to work on a Debian system?

File: doc/install/fast-install-and-first-run.rst (hunk 2 above)
Comment (secnum) [reply doc/install/fast-install-and-first-run.rst]:
```suggestion
   $ echo 'DB_FLOW = "neo4j://neo4j:<new password>@localhost:7474/"' | sudo tee /etc/ivre.conf # Soon unnecessary after adding MongoDB as the flow database in coming IVRE release. 
```

File: doc/install/fast-install-and-first-run.rst (hunk 3 above)
Comment (secnum) [reply doc/install/fast-install-and-first-run.rst]: Apache Web server is not allowed to read this file as www-data. I admit `chmod o+r` is not a clean way.
```suggestion
   # chgrp www-data /usr/local/share/ivre/web/wsgi/app.wsgi
```

File: doc/install/fast-install-and-first-run.rst [hunk 6]
```diff
@@ -94,7 +112,58 @@ Database init, data download & importation
    $ sudo ivre ipdata --download --import-all
 
 The two last step may take a long time to run (40 minutes on a decent
-server), nothing to worry about.
+server), nothing to worry about. To initiate flowcli, Neo4j default 
+password must be changed through its web interface. Connect to Neo4j
+IP address on port 7474 and enter neo4j:neo4j credentials to login
+then change the password. Don't forget to modify ``/etc/ivre.conf``
+accordingly.
```
Comment (secnum) [inline doc/install/fast-install-and-first-run.rst]:
```suggestion
accordingly. This password modification is soon unnecessary after adding MongoDB as the flow database in coming IVRE release. 
```

File: doc/install/fast-install-and-first-run.rst (hunk 4 above)
Comment (secnum) [reply doc/install/fast-install-and-first-run.rst]: Added a comment below.

File: doc/install/fast-install-and-first-run.rst (hunk 5 above)
Comment (secnum) [reply doc/install/fast-install-and-first-run.rst]: I confirm that it works in Debian stable "Buster". Note that the MongoDB official documentation is not up to date for Debian Buster.

File: doc/install/fast-install-and-first-run.rst [hunk 7]
```diff
@@ -94,7 +112,58 @@ Database init, data download & importation
...
+- Zeek/Bro
+- Argus
+- Nfdump
+- Neo4j
+
+::
+
+   $ sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 9DA31620334BD75D9DCB49F368818C72E52529D4
+   $ echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.0.list
+   $ wget -O- "http://download.opensuse.org/repositories/network:/bro/xUbuntu_18.04/Release.key" | sudo apt-key add -
```
Comment (secnum) [inline doc/install/fast-install-and-first-run.rst]:
```suggestion
   $ wget -O- "http://download.opensuse.org/repositories/network:/bro/xUbuntu_18.04/Release.key" | sudo apt-key add - # If you use Ubuntu 18.04 LTS.
```

File: doc/install/fast-install-and-first-run.rst [hunk 8]
```diff
@@ -94,7 +112,58 @@ Database init, data download & importation
...
+   $ echo 'deb http://download.opensuse.org/repositories/network:/bro/xUbuntu_18.04/ /' | sudo tee '/etc/apt/sources.list.d/bro.list'
```
Comment (secnum) [inline doc/install/fast-install-and-first-run.rst]:
```suggestion
   $ echo 'deb http://download.opensuse.org/repositories/network:/bro/xUbuntu_18.04/ /' | sudo tee '/etc/apt/sources.list.d/bro.list' # If you use Ubuntu 18.04 LTS.
   $ wget -O- "http://download.opensuse.org/repositories/network:/bro/Debian_10/Release.key" | sudo apt-key add - # If you use Debian 10.
    $ echo 'deb http://download.opensuse.org/repositories/network:/bro/Debian_10/ /' | sudo tee '/etc/apt/sources.list.d/bro.list' # If you use Debian 10.
```

File: doc/install/fast-install-and-first-run.rst (hunk 1 above)
Comment (p-l-) [reply doc/install/fast-install-and-first-run.rst]: So this can be removed?

File: doc/install/fast-install-and-first-run.rst (hunk 2 above)
Comment (p-l-) [reply doc/install/fast-install-and-first-run.rst]: #685 has been merged, this can be removed.

File: doc/install/fast-install-and-first-run.rst (hunk 3 above)
Comment (p-l-) [reply doc/install/fast-install-and-first-run.rst]: No I think it is OK to `chmod +r` for such a file, but I don't understand how it could end up without those rights, since this file should be world readable.

File: doc/install/fast-install-and-first-run.rst (hunk 4 above)
Comment (p-l-) [reply doc/install/fast-install-and-first-run.rst]: Same, #685 has been merged.

File: doc/install/fast-install-and-first-run.rst (hunk 5 above)
Comment (p-l-) [reply doc/install/fast-install-and-first-run.rst]: OK.

File: doc/install/fast-install-and-first-run.rst (hunk 6 above)
Comment (p-l-) [reply doc/install/fast-install-and-first-run.rst]: This is unnecessary.

File: doc/install/fast-install-and-first-run.rst (hunk 7 above)
Comment (p-l-) [reply doc/install/fast-install-and-first-run.rst]: I'm not sure, why would this be needed?

**Comment by p-l-** [comment]
You can update the PR by adding more commits to your branch and pushing it.

File: doc/install/fast-install-and-first-run.rst (hunk 1 above)
Comment (secnum) [reply doc/install/fast-install-and-first-run.rst]: Yes

File: doc/install/fast-install-and-first-run.rst (hunk 2 above)
Comment (secnum) [reply doc/install/fast-install-and-first-run.rst]: Okay

File: doc/install/fast-install-and-first-run.rst (hunk 3 above)
Comment (secnum) [reply doc/install/fast-install-and-first-run.rst]: Without this chmod, we see wsgi errors in Apache's error.log.

File: doc/install/fast-install-and-first-run.rst (hunk 6 above)
Comment (secnum) [reply doc/install/fast-install-and-first-run.rst]: Okay

File: doc/install/fast-install-and-first-run.rst (hunk 7 above)
Comment (secnum) [reply doc/install/fast-install-and-first-run.rst]: This is necessary to use last Zeek/Bro version. In addition, it is necessary to distinguish Ubuntu and Debian repositories since there is a problem of dependencies (breaks apt).

File: doc/install/fast-install-and-first-run.rst (hunk 4 above)
Comment (secnum) [reply doc/install/fast-install-and-first-run.rst]: Okay

File: doc/install/fast-install-and-first-run.rst [hunk 9]
```diff
@@ -5,50 +5,78 @@ This file describes the steps to install IVRE, run the first scans and
...
-   $ echo 'APT::Default-Release "stable";' | \
-   >   sudo tee /etc/apt/apt.conf.d/99defaultrelease
-   $ echo 'deb http://deb.debian.org/debian testing main' | \
-   >   sudo tee -a /etc/apt/sources.list
-   $ sudo apt-get update
-
-Setup
------
+   $ sudo apt install -y python3-pip python3-dev apache2 libapache2-mod-wsgi-py3 dokuwiki
+   $ sudo -H pip3 install bottle pycrypto future pymongo
```
Comment (p-l-) [inline doc/install/fast-install-and-first-run.rst]: Is there a reason for not using the Debian packages python3-bottle, python3-future, python3-pymongo and python3-crypto?

**Comment by p-l-** [comment]
Please have a look at the error from Travis.
